src/nidigital/metadata/functions_addon.py

```diff
@@ -142,3 +142,71 @@
         ],
     },
 }
+
+functions_additional_fetch_history_ram_cycle_information = {
+    'FancyFetchHistoryRAMCycleInformation': {
+        'python_name': 'fetch_history_ram_cycle_information',
+        'codegen_method': 'python-only',
+        'method_templates': [
+            {
+                'documentation_filename': 'default_method',
+                'method_python_name_suffix': '',
+                'session_filename': 'fancy_fetch_history_ram_cycle_information',
+            }
+        ],
+        'documentation': {
+            'description': '\nReturns the pattern information acquired for the specified cycles.\n\nIf the pattern is using the edge multiplier feature, cycle numbers represent tester cycles, each of which may\nconsist of multiple DUT cycles. When using pins with mixed edge multipliers, pins may return\nNIDIGITAL_VAL_PIN_STATE_NOT_ACQUIRED for DUT cycles where those pins do not have edges defined.\n'
+        },
+        'parameters': [
+            {
+                'direction': 'in',
+                'name': 'vi',
+                'type': 'ViSession'
+            },
+            {
+                'direction': 'in',
+                'documentation': {
+                    'description': 'Site on which to retrieve History RAM data. Specify site as a string in the form of siteN,\nwhere N is the site number. The VI returns an error if more than one site is specified.'
+                },
+                'name': 'site',
+                'type': 'ViConstString'
+            },
+            {
+                'direction': 'in',
+                'documentation': {
+                    'description': 'Pins for which to retrieve History RAM data. If empty, the pin list from the pattern\ncontaining the start label is used. Call get_pattern_pin_list or get_pattern_pin_names with the start\nlabel to retrieve the pins associated with the pattern burst.'
```

Comment:
Reference methods as `niDigital_BlahBlah` so they will be turned into a link in help.

Reply:
Done